### ISSUE TITLE
Crowdin Language Translations

### ISSUE THREAD

**Issue by haraldpdl** [osCommerce/oscommerce2#486]
Research how to best work with Crowdin for language translations.

**Comment by Gergely** [comment]
Language files saved in Crowdin directory language structure. Oscommerce doesnt have Crowdin supported language codes.

**iso_639_1** vs **crowdin_code**

https://support.crowdin.com/api/language-codes/

For example Spanish and Hungarian directory structure in Crowdin is found:
**es-ES\
hu\**

**Comment by haraldpdl** [comment]
The code for the standard English languages matches: en

We could standardize on "en-US" however "en" is an acceptable crowdin code.

Additional languages can use their specific code.

**Comment by Gergely** [comment]
I have planned english as source base but another directories should be renamed when zip import assign to reflected core language.

I see that we can get language codes when used API from request but download/upload zip export/import wont work without associated language codes.

**Comment by haraldpdl** [comment]
Right, but once the admin page is ready it can create an export package based on the language code, or accept an import package that matches the language code. This has no relation with the language directory used in core.
